packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -34,6 +34,56 @@ typedef ScrollableWidgetBuilder = Widget Function(
   ScrollController scrollController,
 );
 
+class DraggableScrollableSheetController {
+  DraggableScrollableSheetController();
+
+  late _DraggableScrollableSheetScrollController _scrollController;
```

Comment:
This should be nullable along with a `!= null` assert in each relevant function (eg `animateTo`) with a useful error message. When you get around to testing, make sure to include some tests that ensure exceptions are thrown when they should be. There are already some draggable sheet tests you can copy from for this.